src/config/util.js

```diff
@@ -42,7 +42,9 @@ function requireFromString(src, args) {
 
     return mod;
   } catch (err) {
-    console.warn('Failed to import ${src}');
+    if (!(err.message || '').match(/Cannot find module \'@iopipe\/config\'/)) {
```

Comment:
@coreylight Why are we silencing this?

Reply:
Because will happen with any install of any flavor of the agent that does not have `@iopipe/config` available. We have a known `try/catch` failure state for `@iopipe/config`

Reply:
@coreylight Upon further examination, maybe removing `@iopipe/config` as the default in `extends` and setting it explicitly in the `package.json` for `@iopipe/iopipe` is a better way to do.

Reply:
Yes, I think that was the plan I had in my head originally, I think we can do that